Amplify/Categories/DataStore/DataStoreCategory+Behavior.swift

```diff
@@ -18,6 +18,19 @@ extension DataStoreCategory: DataStoreBaseBehavior {
         plugin.query(modelType, byId: id, completion: completion)
     }
 
+    public func query<M: Model>(_ modelType: M.Type,
```

Comment:
The new changes in `DataSttoreCategory+Behavior` somehow doesn't compile with Xcode 14.

<img width="730" alt="image" src="https://user-images.githubusercontent.com/10602282/178066709-513a74f5-f45d-4b87-aefc-1582ec87cbb4.png">

Reply:
Thanks for finding this, adding fix here: https://github.com/aws-amplify/amplify-ios/pull/2067